src/app.js

```diff
@@ -20,7 +20,7 @@ const CANARY_ORIGIN = process.env.CANARY_ORIGIN;
 /* eslint-disable import/first */
 import * as utils from './utils/index.js';
 import { addAsync } from './async.js';
-import { Forbidden, NotFound, Unauthorized } from './httpErrors.js';
+import { Forbidden, NotFound, Unauthorized, NotAcceptable } from './httpErrors.js';
```

Comment:
Remember to drop this if you cherry-pick the suggestion in https://github.com/nextstrain/nextstrain.org/pull/618/files#r1012106232.